articles/virtual-desktop/set-up-scaling-script.md

```diff
@@ -78,7 +78,9 @@ First, you'll need an Azure Automation account to run the PowerShell runbook. He
 3. Run the following cmdlet to download the script for creating the Azure Automation account:
 
      ```powershell
-     Invoke-WebRequest -Uri "https://raw.githubusercontent.com/Azure/RDS-Templates/master/wvd-templates/wvd-scaling-script/createazureautomationaccount.ps1" -OutFile "your local machine path\ createazureautomationaccount.ps1"
+     Set-Location -Path "c:\temp"
+$uri = "https://raw.githubusercontent.com/Azure/RDS-Templates/master/wvd-templates/wvd-scaling-script/createazureautomationaccount.ps1"
+Invoke-WebRequest -Uri $uri -OutFile ".\createazureautomationaccount.ps1"
```

Comment:
@michawets : Can you indent these so that they all appear together in the same code block?

Reply:
Hi @ChristianMontoya , I totally forgot the indentation 😊 Should be fixed now